content/en-us/reference/engine/classes/Instance.yaml

```diff
@@ -1073,6 +1073,12 @@ methods:
       
       Note that this event will not pass any arguments to a connected function,
       so the value of the changed property must be read directly within a script.
+
+      <Alert severity="error">
+      Like `Class.Instance.Changed|Changed`, the event returned by this method does not fire for physics-related changes, 
+      like when the `Datatype.CFrame`, `Velocity`, `RotVelocity`, `Position`, `Orientation`
+      and `Datatype.CFrame` properties of a `Class.BasePart` change due to gravity.
+      </Alert>
```

Comment:
```suggestion
      Like `Class.Instance.Changed|Changed`, the event returned by this method does not
      fire for physics-related changes, such as when the `Class.BasePart.CFrame|CFrame`,
      `Class.BasePart.AssemblyLinearVelocity|AssemblyLinearVelocity`,
      `Class.BasePart.AssemblyAngularVelocity|AssemblyAngularVelocity`,
      `Class.BasePart.Position|Position`, or `Class.BasePart.Orientation|Orientation`
      properties of a `Class.BasePart` change due to gravity or other physical forces.
```

Reply:
Changing the cited properties to the exact existing ones on BasePart. Also removing the <Alert> tags since those currently are not supported for API doc pages.